Update vusb (#1724)
saucelabs/sauce-docs: docs/mobile-apps/features/virtual-usb.md

{label: 'iOS', value: 'iOS'},
]}>

<!-- prettier-ignore-start -->
<TabItem value="Android">

The expected output will be a **port number**, which you'll need when you want to connect the device to ADB (see Step 7).

```java
localhost:7000  online
```

</TabItem>
<TabItem value="iOS">

```java
localhost:-1  online
```

After this, vUSB needs to have access to the usbmuxd socket on your computer. This is a socket that is used by XCODE (Apple's developer tools) to communicate with iOS devices. By default it is connected to your local USB port, but since we are using a remote device, vUSB need to move the socket to a different location. To be able to do this vUSB needs to have read and write access to the socket which is located at `/var/run/usbmuxd`.

> If you are running vUSB as a non-root user you will need to provide the password for the root user and an Apple system notification popup will be shown, see below

Reviewer: ```suggestion
If you are running vUSB as a non-root user you will need to provide the password for the root user and an Apple system notification popup will appear:
```

localhost:-1  online
```

After this, vUSB needs to have access to the usbmuxd socket on your computer. This is a socket that is used by XCODE (Apple's developer tools) to communicate with iOS devices. By default it is connected to your local USB port, but since we are using a remote device, vUSB need to move the socket to a different location. To be able to do this vUSB needs to have read and write access to the socket which is located at `/var/run/usbmuxd`.

> If you are running vUSB as a non-root user you will need to provide the password for the root user and an Apple system notification popup will be shown, see below

Author: ```suggestion
If you are running vUSB as a non-root user you will need to provide the password for the root user and an Apple system notification popup will appear:
```

> After this, vUSB needs to have access to the usbmuxd socket on your computer. This socket is used by XCODE (Apple's developer tools) to communicate with iOS devices. By default, it is connected to your local USB port, but since we are using a remote device, vUSB needs to move the socket to a different location. To do this, vUSB needs to have read and write access to the socket which is located at `/var/run/usbmuxd`.
> 
> If you are running vUSB as a non-root user you will need to provide the password for the root user and an Apple system notification popup will appear :
> 
> <img src={useBaseUrl('img/virtual-usb/vusb-usbmuxd.png')} alt="vUSB usbmuxd popup" width="370" />
> 
> If you are running vUSB as a root user, you will not be prompted for a password and the socket will be moved automatically. You will see the following message in your server logs:

Reviewer: ```suggestion
After this, vUSB needs to have access to the usbmuxd socket on your computer. This socket is used by XCODE (Apple's developer tools) to communicate with iOS devices. By default, it is connected to your local USB port, but since we are using a remote device, vUSB needs to move the socket to a different location. To do this, vUSB needs to have read and write access to the socket which is located at `/var/run/usbmuxd`.

If you are running vUSB as a non-root user you will need to provide the password for the root user and an Apple system notification popup will appear:

<img src={useBaseUrl('img/virtual-usb/vusb-usbmuxd.png')} alt="vUSB usbmuxd popup" width="370" />

If you are running vUSB as a root user, you will not be prompted for a password and the socket will be moved automatically. You will see the following message in your server logs:
```

> If you are running vUSB as a root user you will not be prompted for a password and the socket will be moved automatically. You will see the following message in your server logs:

Reviewer: ```suggestion
After this, vUSB needs to have access to the usbmuxd socket on your computer. This socket is used by XCODE (Apple's developer tools) to communicate with iOS devices. By default, it is connected to your local USB port, but since we are using a remote device, vUSB needs to move the socket to a different location. To do this, vUSB needs to have read and write access to the socket which is located at `/var/run/usbmuxd`.

If you are running vUSB as a non-root user you will need to provide the password for the root user and an Apple system notification popup will appear :

<img src={useBaseUrl('img/virtual-usb/vusb-usbmuxd.png')} alt="vUSB usbmuxd popup" width="370" />

If you are running vUSB as a root user, you will not be prompted for a password and the socket will be moved automatically. You will see the following message in your server logs:
```

```bash
11:13:12.347 INFO com.saucelabs.vusb.client.server.usbmuxd.SocketMover - The socket at /var/run/usbmuxd needs to be moved
11:13:12.347 INFO com.saucelabs.vusb.client.server.usbmuxd.SocketMover - This will require administrator privileges!
```

> Make sure you're logged into your Sauce Labs account prior to clicking the link above. After doing so, you'll see: 1) an Apple system notification popup, where you'll need to provide Touch ID or password authentication; and 2) information returned in your server logs similar to the example below.

Reviewer: ```suggestion
Make sure you're logged into your Sauce Labs account prior to clicking the link above. After doing so, you'll see an Apple system notification popup, where you'll need to provide Touch ID or password authentication, and information returned in your server logs similar to the example below.
```
